system/Console.php

```diff
@@ -1,26 +1,27 @@
 <?php
+
 namespace System;
 
 class Console
 {
-  public $namespace;
+    public $namespace;
 
-  public function __construct () 
-  {
-    if (PHP_SAPI !== 'cli' || @$_SERVER['argv'][0] !== 'console.php' || @$_SERVER['argv'][1] !== 'console') {
-      header('HTTP/1.0 404 Not Found', true, 404);
-      exit;
+    public function __construct()
```

Comment:
__[PHPMD]__ __construct accesses the super-global variable $_SERVER.

<sub>posted by Sider</sub>